intake_xarray/tests/test_remote.py

```diff
@@ -45,6 +47,7 @@ def test_list(data_server):
 
 
 def test_open_rasterio(data_server):
+    pytest.importorskip("resterio")
```

Comment:
Should this be rasterio?

Reply:
That does seem likely... It would not have made a difference on CI, since rasterio is included in the env.